addons/website/models/res_config_settings.py

```diff
@@ -37,7 +37,7 @@ def _default_website(self):
     cdn_activated = fields.Boolean(related='website_id.cdn_activated', readonly=False)
     cdn_url = fields.Char(related='website_id.cdn_url', readonly=False)
     cdn_filters = fields.Text(related='website_id.cdn_filters', readonly=False)
-    auth_signup_uninvited = fields.Selection(compute="_compute_auth_signup", inverse="_set_auth_signup")
+    auth_signup_uninvited = fields.Selection(related="website_id.auth_signup_uninvited", readonly=False)
```

Comment:
not checked in detail, but seems like a revert of https://github.com/odoo-dev/odoo/commit/cb9f48d3a91d15c9004201beada26ad570f9d016 from @mart-e

Reply:
Maybe, good to know anyway.  It'll be considered when I (we) work on this again (it was just a rebase after the partial merge of the PR content by TDE)